src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/VirtualDynamoTableDescriptionImpl.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.dynamodbv2.mt.mappers.metadata;
+
+import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
+import com.salesforce.dynamodbv2.mt.repo.MtTableDescription;
+
+public class VirtualDynamoTableDescriptionImpl extends DynamoTableDescriptionImpl
+    implements VirtualDynamoTableDescription {
+
+    private final boolean isMultitenant;
+
+    public VirtualDynamoTableDescriptionImpl(CreateTableRequest createTableRequest, boolean isMultitenant) {
+        super(createTableRequest);
+        this.isMultitenant = isMultitenant;
+    }
+
+    public VirtualDynamoTableDescriptionImpl(MtTableDescription tableDescription) {
```

Comment:
MtTableDescription extends TableDescription, with an extra flag `isMultitenant` (and more properties in the future), and that's what we store in the table description repo now. This should make us still be able to parse existing TableDescription records.